modules/roks-ingress/main.tf

```diff
@@ -205,7 +207,7 @@ data "ibm_is_network_acl_rules" "alb_acl_rules" {
 resource "ibm_is_network_acl_rule" "alb_https_req" {
   count       = var.cluster_zone_count
   network_acl = data.ibm_is_network_acl.alb_acl.id
-  before      = local.cluster_acl_deny_rule
+  before      = local.cluster_acl_deny_rule != null ? local.cluster_acl_deny_rule : ""
```

Comment:
I don't think you are ever going to wan't to pass `""` as a value to an attribute. Did you mean `null` ?